inst/tws/tests/testthat/test-shinytest2.R

```diff
@@ -0,0 +1,24 @@
+library(shinytest2)
+
+test_that("{shinytest2} recording: tws_screenshot", {
+  app <- AppDriver$new(
+    variant = platform_variant(), name = "tws_screenshot",
+    height = 823, width = 1459
+  )
+  app$expect_screenshot()
+})
+
+test_that("{shinytest2} recording: tws_download", {
+  app <- AppDriver$new(
+    variant = platform_variant(), name = "tws_download", height = 823,
+    width = 1459
+  )
+  app$set_window_size(width = 1459, height = 766)
```

Comment:
I think we should add some snapshot tests, recording the values of inputs and outputs. Here's an example:

```
test_that("snapshotting inputs", {

  app <- AppDriver$new(name = 'blah')

  app$expect_values(
    input = c(
      'table_with_settings-downbutton-file_format',
      'table_with_settings-downbutton-lpp',
      'table_with_settings-downbutton-pagination_switch'
    )
  )

  file_name <- app$get_value(input = 'table_with_settings-downbutton-file_name')
  file_name <- sub('_\\d{6}$', '', file_name) # remove time
  expected_file_name <- paste0("table_", strftime(Sys.Date(), format = "%Y%m%d"))
  testthat::expect_equal(file_name, expected_file_name)

  app$expect_text("table")
  app$stop()

})
```

Try adding this and looking at the snapshot files ('.json' and '.md'). Snapshotting the values like this is more robust than screenshots. And you can do any manipulation needed on the object (e.g. file name).